psake-build-helpers.ps1

```diff
@@ -0,0 +1,44 @@
+function Set-Project-Properties($versionPrefix) {
+    write-host "$versionPrefix"
+
+    set-regenerated-file "$pwd/Directory.Build.props" @"
+<Project>
+    <PropertyGroup>
+        <VersionPrefix>$versionPrefix</VersionPrefix>
+    </PropertyGroup>
+</Project>
+"@
+}
+
+function Get-Copyright {
+    $date = Get-Date
+    $year = $date.Year
+    $copyrightSpan = if ($year -eq $yearInitiated) { $year } else { "$yearInitiated-$year" }
+    return "� $copyrightSpan $owner"
+}
+
+function Publish-Project {
+    $project = Split-Path $pwd -Leaf
+    Write-Host "Publishing $project"
+    dotnet publish --configuration $configuration --no-restore --output $publish/$project /nologo
```

Comment:
I have a newer version of this script where `--no-restore` is replaced with `--no-build` assuming you've already done a compile / test cycle.